include/ggml/ggml.h

```diff
@@ -201,6 +201,9 @@
 #define GGML_MAX_NAME          48
 #define GGML_DEFAULT_N_THREADS 4
 
+#define GGML_EXIT_SUCCESS 0
+#define GGML_EXIT_ABORTED 1
+
```

Comment:
Probably move into `ggml.c`

Reply:
Could we have a public return status for the `ggml_graph_compute_with_abort` to see whether it got aborted?

Reply:
Yes. If so - we cab keep the defines in the header